doc/rst/language/spec/statements.rst

```diff
@@ -267,6 +267,10 @@ the right-hand-side; or reading from the left-hand-side and storing the
 result in the variable on the right-hand-side. This operator can be
 chained with other I/O operator calls.
 
+.. warning::
+
+   The I/O operator is deprecated.
```

Comment:
Other places in the docs seem to use a "note" for saying that the operator is deprecated: 
![Screen Shot 2022-09-07 at 11 55 33 AM](https://user-images.githubusercontent.com/4361282/188956095-5a0809a3-66a4-4373-8b4f-4ab4743d9f23.png)
Does the consistency matter at all?

Reply:
Good catch, I'll update that in a moment.